MILESTONE-1.md

```diff
@@ -40,11 +40,11 @@ The authenticator contract address that is used to verify the addresses of voter
 - **next_proposal_nonce** = 1  
 An increasing nonce to make the proposal id unique.
 
-- **proposals** = TBD  
-A mapping of proposals by ids with required information: execution_hash, start, end.
+- **proposals** = { 1: { start: 123.., end: 456.., execution_hash: '0x123...', scores: { 0: 123, 1: 45, 3: 6 }]
```

Comment:
Having a list or a mapping inside a storage_var's value is complicated. This is because you can't simply have a pointer in there because the segment it points to won't be stored for future calls to this contract
The better way to do this here is to have two mappings:
1. A map between a proposal_id to a struct containing start, end and execution_hash. 
2. A map between a proposal_id and choice to number of votes

Reply:
I see thanks, I'll update the doc